slangpy/testing/helpers.py

```diff
@@ -44,6 +45,9 @@
 else:
     raise RuntimeError("Unsupported platform")
 
+# If live object tracking is supported, enable leak tracking
+LEAK_TRACKING_ENABLED = hasattr(spy.Object, "report_live_objects")
```

Comment:
Should we move all the leak checking specifics into a new `leak_tracking.py` module?